docs/process/v2_to_v3_environment_variables.md

```diff
@@ -26,11 +26,9 @@ Below is a list of affected variables and their new names.
 | `PERCONA_TEST_CHECKS_PUBLIC_KEY`              |                                            | Removed in PMM v3, use `PMM_DEV_PERCONA_PLATFORM_PUBLIC_KEY` |
 | `PERCONA_TEST_NICER_API`                      | `PMM_NICER_API`                            |                                                              |
 | `PERCONA_TEST_PMM_CLICKHOUSE_ADDR`            | `PMM_CLICKHOUSE_ADDR`                      |                                                              |
-| `PERCONA_TEST_PMM_CLICKHOUSE_BLOCK_SIZE`      | `PMM_CLICKHOUSE_BLOCK_SIZE`                |                                                              |
```

Comment:
I think that since the variables still exist in v2, it's better we keep them here and mention that they're now removed in v3 in the comments column.

Reply:
Added back with description.